common/utils.js

```diff
@@ -3,18 +3,6 @@ export function getUniqueString (prefix = 'dt') {
   return `${prefix}${UNIQUE_ID_COUNTER++}`;
 }
 
-/**
- * Transform a string from kebab-case to PascalCase
- * @param string
- * @returns {string}
- */
-export function kebabCaseToPascalCase (string) {
-  return string?.toLowerCase()
-    .split('-')
-    .map(word => word.charAt(0).toUpperCase() + word.slice(1))
-    .join('');
-}
-
```

Comment:
I think your linter did this, right? this is being used on icon component IIRC

Reply:
I get this...

```
import { kebabCaseToPascalCase } from '../../common/utils.js';
         ^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Named export 'kebabCaseToPascalCase' not found. The requested module '../../common/utils.js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '../../common/utils.js';
const { kebabCaseToPascalCase } = pkg;
```

Do we need to rename this to utils.mjs since package.json is not type: "module" anymore?

Reply:
Changed this as I mentioned above, seems to have fixed it.